partcad-cli/src/partcad_cli/click/commands/config.py

```
@@ -0,0 +1,12 @@
+import rich_click as click
+from partcad.logging import Process, info, debug
```

Ruff (0.8.2)

`partcad.logging.Process` imported but unused

Remove unused import: `partcad.logging.Process`

(F401)